src/components/Event.js

```diff
@@ -88,22 +88,12 @@ class Event extends Component {
               <div className="icon"><DollarIcon /></div>
               <p>{ `${this.props.event.price} Yen`}</p>
               <div className="grow" />
-              { this.props.userProfile ?
-                <Stripe
-                  label={'Reserve'}
-                  token={(token) => { this.onToken(token); }}
-                  stripeKey="pk_test_6pRNASCoBOKtIshFeQd4XMUh"
-                  currency="JPY"
-                  name="Live Jazz Co."
-                  image="../logo.png"
-                  description={`1 ticket for ${this.props.event.name}`}
-                  amount={2000}
+              <Link to={'/pay'}>
+                <RaisedButton
+                  className="mui-button"
+                  label="Reserve"
```

Comment:
@bill-feng Seems like the primary color for reserve button is gone :( Do you think you can put the style back?

Reply:
Yea the whole button is gone now. We need to do styling separately if we want!